extensions/wrapper/wrapper-common-api/src/main/java/de/sovity/edc/ext/wrapper/api/common/model/UiAssetCreateRequest.java

```diff
@@ -74,6 +75,36 @@ public class UiAssetCreateRequest {
     @Schema(description = "Transport Mode", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
     private String transportMode;
 
+    @Schema(description = "The publisher and sovereign can be distinct", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private String sovereign;
```

Comment:
We need suffixes to distinguish what these fields mean. Is it the Homepage URL, is it the Legal Name, etc.